paparazzi/api/paparazzi.api

```diff
@@ -0,0 +1,255 @@
+public final class app/cash/paparazzi/DeviceConfig {
+	public static final field $stable I
+	public static final field Companion Lapp/cash/paparazzi/DeviceConfig$Companion;
+	public static final field GALAXY_WATCH4_CLASSIC_LARGE Lapp/cash/paparazzi/DeviceConfig;
+	public static final field NEXUS_10 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field NEXUS_4 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field NEXUS_5 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field NEXUS_5_LAND Lapp/cash/paparazzi/DeviceConfig;
+	public static final field NEXUS_7 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field NEXUS_7_2012 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_2 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_2_XL Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_3 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_3A Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_3A_XL Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_3_XL Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_4 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_4A Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_4_XL Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_5 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_6 Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_6_PRO Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_C Lapp/cash/paparazzi/DeviceConfig;
+	public static final field PIXEL_XL Lapp/cash/paparazzi/DeviceConfig;
+	public static final field WEAR_OS_SMALL_ROUND Lapp/cash/paparazzi/DeviceConfig;
+	public static final field WEAR_OS_SQUARE Lapp/cash/paparazzi/DeviceConfig;
+	public fun <init> ()V
+	public fun <init> (IIIILcom/android/resources/ScreenOrientation;Lcom/android/resources/UiMode;Lcom/android/resources/NightMode;Lcom/android/resources/Density;FLcom/android/resources/LayoutDirection;Ljava/lang/String;Lcom/android/resources/ScreenRatio;Lcom/android/resources/ScreenSize;Lcom/android/resources/Keyboard;Lcom/android/resources/TouchScreen;Lcom/android/resources/KeyboardState;ZLcom/android/resources/Navigation;Lcom/android/resources/ScreenRound;Ljava/lang/String;)V
```

Comment:
Wanted to highlight that you might be able to hide the SnapshotHandler implementations IF you make `determineHandler` public (which it should be anyway, because it's tricky to replicate that logic otherwise.)

Reply:
Yea, I considered this, but it seems to be that the conditional logic varies by consumer, making `determineHandler` less useful than exposing the implementations directly (for now...).